gp-templates/translations.php

```diff
@@ -183,7 +183,12 @@
 		if ( 'rejected' == $status ) continue;
 ?>
 	<div class="box status-<?php echo $status; ?>"></div>
-	<div><?php echo $status; ?></div>
+	<div>
+		<?php if ('current' == $status ) { _e( 'Current', 'glotpress' ); } ?>
+		<?php if ('waiting' == $status ) { _e( 'Waiting', 'glotpress' ); } ?>
+		<?php if ('fuzzy' == $status ) { _e( 'Fuzzy', 'glotpress' ); } ?>
+		<?php if ('old' == $status ) { _e( 'Old', 'glotpress' ); } ?>
+	</div>
```

Comment:
Are there more than those 4 statuses? Only warnings are shown separate and translated.